app/controllers/publish/courses/withdrawals_controller.rb

```diff
@@ -15,7 +15,7 @@ def edit
       def update
         authorize(provider)
 
-        @course_withdrawal_form = CourseWithdrawalForm.new(course, params: withdrawal_params)
+        @course_withdrawal_form = CourseWithdrawalForm.new(course)
```

Comment:
Do we want the whole thing behind the feature flag? I don't see any reason to do so.

Reply:
Agree, I don't think there is much of a need to put it behind the feature flag